src/execCmd.ts

```diff
@@ -114,7 +114,7 @@ const getExitCodeError = (cmd: string, expectedCode: number, output: ShellString
  *
  * If the cli is 'inherit', the executable preference order is:
  *    1. TESTKIT_EXECUTABLE_PATH env var
- *    2. `bin/dev.js` (default)
+ *    2. `bin/run.js` (default)
```

Comment:
I think it would be cool to have a bin/dev option.  Like bin/run could be the default, but bin/dev would be an option for `cli`.

Why?  Maybe we'd need to use dev-plugins somewhere, or maybe we'd just want to verify that bin/dev is behaving well under changes, or maybe someone doesn't like `yarn build` before re-running NUTs locally?  Or maybe we are interested in comparing perf?

Ideally this would be behind an env, since testkit has a lot of those.

Reply:
that would also provide a quick "path back" for repos that do need their devPlugins to be able to stay on this version.